gql/types.go

```diff
@@ -51,3 +54,22 @@ func (dt *DateTime) UnmarshalGQL(v interface{}) error {
 func (dt DateTime) MarshalGQL(w io.Writer) {
 	graphql.MarshalTime(time.Time(dt)).MarshalGQL(w)
 }
+
+// MarshalMachineState helps mapping sabakan.MachineState with GraphQL enum.
```

Comment:
This and `UnmarshalMachineState` are necessary to map `sabakan.MachineState` to
GraphQL `MachineState` enum type.